sdcm/commit_log_check_thread.py

```diff
@@ -176,13 +178,26 @@ def run_thread(self):
                 message=f"CommitLogCheckThread failed: {exc.__repr__()} with traceback {trace}").publish()
         self.log.debug("CommitLogCheckThread finished")
 
+    @staticmethod
+    def get_commit_log_directory_max_exceed_time(prometheus_response):
+        # get max time diff from response per shard(line in response)
+        exceed_time_list = []
+        for line in prometheus_response:
+            time_list = [item[0] for item in line["values"]]
+            time_list.sort()
+            print(time_list[-1] - time_list[0])
+            exceed_time_list.append(time_list[-1] - time_list[0])
+            exceed_time_list.sort()
```

Comment:
not sure why sorting it on every round ?, and not at the end when returning ?

Reply:
you are right